desc/objectives/objective_funs.py

```diff
@@ -33,21 +34,38 @@ class ObjectiveFunction(IOAble):
         Method for computing Jacobian matrices. "batched" uses forward mode, applied to
         the entire objective at once, and is generally the fastest for vector valued
         objectives, though most memory intensive. "blocked" builds the Jacobian for each
-        objective separately, using each objective's preferred AD mode. Generally the
-        most efficient option when mixing scalar and vector valued objectives.
-        "looped" uses forward mode jacobian vector products in a loop to build the
-        Jacobian column by column. Generally the slowest, but most memory efficient.
+        objective separately, using each objective's preferred AD mode (and
+        each objective's `chunk_size`). Generally the most efficient option when mixing
+        scalar and vector valued objectives. "looped" uses forward mode jacobian vector
```

Comment:
Question about looped: I've tried using looped mode in an optimization where I increase the number of optimization variables at each step, and at a later step it crashed with an out of memory error. That is expected for batched mode, but is it expected for looped? I thought looped would use about that same amount of memory regardless of the number of optimization variables

Reply:
When was this? I know a while ago looped was actually broken, though I think it's fixed now. It could also be to some recompilation memory leaks that should have been fixed in #1043

Reply:
removed `looped` (with deprecation warning)